
char * FUN_00623340(void)

{
  return "std::bad_exception";
}

